gradle-baseline-java-config/resources/checkstyle/checkstyle-suppressions.xml

```diff
@@ -19,6 +19,7 @@
     <suppress files="\.(bdr|eot|gif|gzip|jar|json|otf|png|svg|ttf|woff|zip)$" checks="NewlineAtEndOfFile" />
 
     <!-- Generated code should not be subjected to checkstyle. -->
+    <suppress files="[/\\].*[/\\]generated[/\\]" checks="." />
     <suppress files="[/\\].*[/\\]generated_src[/\\]" checks="." />
```

Comment:
We should stick to maven convention unless there's a really good reason, and maven dictates that src is the stuff that lives in source control.